trade-finance-manager-api/src/v1/controllers/deal-add-tfm-data/index.api-test.js

```diff
@@ -20,12 +20,13 @@ describe('deal submit - add TFM data', () => {
       tfm: {},
     });
 
-    const result = await addTfmDealData(mockDeal);
+    const result = await addTfmDealData(mockDeal, MOCK_PORTAL_USERS[0]);
 
     const expected = {
       ...mockDeal,
       tfm: {
-        ...generateDateReceived(),
+        dateReceived: expect.any(String),
```

Comment:
waas this changed as it was flakey?

Reply:
Yep, it was one of the classic `expect an exact time in seconds` that occasionally is a bit slow and fails

Reply:
fwiw i usually like using fake timers for this (note the `doNotFake: ['nextTick']` might not be needed here)

```
  beforeAll(async () => {
    // Not faking next tick is required for database interaction to work
    jest.useFakeTimers({
      doNotFake: ['nextTick'],
    });
  });

  afterAll(async () => {
    jest.useRealTimers();
  });

```